services/integration/src/controllers/project.js

```diff
@@ -5,22 +5,22 @@ const create = asyncHandler(async (req, res) => {
   const project = new Project(req.body);
 
   await project.save();
-
-  res.send('Project Created successfully');
+  
+  res.send({message :'Project Created successfully'});
 });
 
 const get = asyncHandler(async (req, res) => {
   const projectId = req.params.id;
 
   const project = await Project.findById(projectId);
 
-  res.send(project);
+  res.send({message : project});
 });
 
 const find = asyncHandler(async (req, res) => {
   const project = await Project.find({});
 
-  res.send(project);
+  res.send({message: project});
```

Comment:
This is not the right way, a message is a text string. Please revert those changes.